libraries/adaptive-expressions/src/expressionProperties/intExpression.ts

```diff
@@ -14,6 +14,10 @@ import { Expression } from '../expression';
  * String values are always interpreted as an expression, whether it has '=' prefix or not.
  */
 export class IntExpression extends ExpressionProperty<number> {
+    /**
+     * Initializes a new instance of the `ArrayExpression<T>` class.
+     * @param value An `int` or `string` expression which resolves to a `int`.
```

Comment:
Should we use `int` in here?

Reply:
I corrected it to "int `number`" and "float `number`" since those descriptors are relevant for the to describe JSON conversion units.